build.sbt

```diff
@@ -10,8 +10,11 @@ scalaVersion := "2.10.4"
 
 libraryDependencies += "org.apache.hadoop" % "hadoop-client" % "1.0.4"
 
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "1.2.0"
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "1.3.0-SNAPSHOT"
```

Comment:
Maybe this could depend on the 1.3.0 RC1 published to the Apache staging repo instead, since that won't require publishLocal?